cms/static/js/index.js

```diff
@@ -51,12 +51,19 @@ define(["domReady", "jquery", "underscore", "js/utils/cancel_on_escape", "js/vie
             var org = $newCourseForm.find('.new-course-org').val();
             var number = $newCourseForm.find('.new-course-number').val();
             var run = $newCourseForm.find('.new-course-run').val();
+            var license = {};
+            if(licenseSelector){
```

Comment:
Are you checking to see if the `licenseSelector` variable is defined? Because if so, this will fail with a `ReferenceError` if the variable is not defined. (If the variable is defined, but set to a falsy value like `null`, then it will do what you expect.) If you need to worry about the case where the variable may not be defined, then you're better off using the `typeof` keyword to check that.
```js
if (typeof licenseSelector != "undefined" && licenseSelector) {
```